tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot

```diff
@@ -6,20 +6,65 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 Default Tags
 
 *** Test Cases ***
-Display HTML
+Get Login Page
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/authentication
+    Should contain  ${output}  <title>VCH Admin</title>
+
+While Logged Out Fail To Display HTML
     ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}
+    Should not contain  ${output}  <title>VIC: ${vch-name}</title>
+
+While Logged Out Fail To Get Portlayer Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/port-layer.log
+    Should Not Contain  ${output}  Launching portlayer server
+
+While Logged Out Fail To Get VCH-Init Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/init.log
+    Should not contain  ${output}  reaping child processes
+
+While Logged Out Fail To Get Docker Personality Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/docker-personality.log
+    Should not contain  ${output}  docker personality
+
+While Logged Out Fail To Get Container Logs
+    ${rc}  ${output}=  Run And Return Rc and Output  docker ${params} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${container}=  Run And Return Rc and Output  docker ${params} create busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${container}  Error
+    ${rc}  ${output}=  Run And Return Rc and Output  docker ${params} start ${container}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
```

Comment:
just a simple docker run would be clearer and easier here.

docker run -itd busybox /bin/top or so